types/cesium/index.d.ts

```diff
@@ -2735,6 +2739,7 @@ declare namespace Cesium {
             heightReference?: Property;
             scaleByDistance?: Property;
             distanceDisplayCondition?: Property;
+            disableDepthTestDistance?: number;
```

Comment:
Constructor options for Graphics objects can be either the raw value or a Property.